scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/Configuration.java

```diff
@@ -300,6 +307,18 @@ public String getJavaHome() {
         }
     }
 
+    public List<Integer> getElasticsearchPorts() {
+        if (elasticsearchPorts.isEmpty()) {
+            return Collections.emptyList();
+        }
+        String[] portsRaw = elasticsearchPorts.replace(" ", "").split(",");
+        ArrayList<Integer> portsList = new ArrayList<>(2);
+        for (String port : portsRaw) {
+            portsList.add(Integer.parseInt(port));
```

Comment:
Either here or (better) in `CLIValidators` I would check if the given ports are numbers. Otherwise with incorrect input we'll get "unable to parse" with an exception trace - no clear explanation of the problem